CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
+# 0.6.0
+
+* Enhancement - Renamed default version and branch to `mainline`.
+* Enhancement - Added support for [AWS SSO](https://aws.amazon.com/single-sign-on/) ([#229](https://github.com/awslabs/amazon-ecr-credential-helper/issues/229))
+* Enhancement - Added support for [AWS SDK for Go V2](https://aws.github.io/aws-sdk-go-v2/docs/getting-started/) ([#282](https://github.com/awslabs/amazon-ecr-credential-helper/issues/282))
```

Comment:
What are the benefits of the V2 SDK? It would be better to talk more about end-user features.

Reply:
Take a swing at this in the [latest commit](https://github.com/austinvazquez/amazon-ecr-credential-helper/commit/ec5d4470d362449b61b23b2bda7b3832c3b522e6). The original issue did not mention a list of features for adopting the new SDK revision. However I pulled feature points from the public AWS SDK for GO v2 release documentation. Please review.